datasets/librispeech/_metadata.json

```diff
@@ -0,0 +1,254 @@
+{
+    "@context": {
+        "@vocab": "https://schema.org/",
+        "ml": "http://mlcommons.org/schema/"
+    },
+    "@type": "Dataset",
+    "@language": "en",
+    "name": "LibriSpeech ASR corpus",
+    "url": "http://www.openslr.org/12",
+    "description": "LibriSpeech is a corpus of approximately 1000 hours of 16kHz read English speech, prepared by Vassil Panayotov with the assistance of Daniel Povey. The data is derived from read audiobooks from the LibriVox project, and has been carefully segmented and aligned.\nAcoustic models, trained on this data set, are available at kaldi-asr.org and language models, suitable for evaluation can be found at http://www.openslr.org/11/.\nFor more information, see the paper \"LibriSpeech: an ASR corpus based on public domain audio books\", Vassil Panayotov, Guoguo Chen, Daniel Povey and Sanjeev Khudanpur, ICASSP 2015 (submitted) (pdf)",
+    "distribution": [
+        {
+            "@type": "FileObject",
+            "contentUrl": "https://www.openslr.org/resources/12/dev-clean.tar.gz",
+            "name": "dev-clean.tar.gz",
+            "sha256": "42e2234ba48799c1f50f24a7926300a1",
+            "description": "[337M] development set, \"clean\" speech",
```

Comment:
It feels a bit wrong to me to use contrived descriptions such as "[337M]" for the size in a MLCommons-endorsed example. Should we put this info in another field?

Again, I can see the merit of the current solution, because it's a direct copy of the original description.

What are your thought?